man/tpm2_policyauthorize.1.md

```diff
@@ -0,0 +1,85 @@
+% tpm2_policyauthorize(1) tpm2-tools | General Commands Manual
+%
+% AUGUST 2018
+
+# NAME
+
+**tpm2_policyauthorize**(1) - Generates/ Creates a policy event that authorizes 
+a policy digest from TPM policy events.
+
+# SYNOPSIS
+
+**tpm2_policyauthorize** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_policyauthorize**(1) generates a policy Authorize event with the TPM. It 
+expects a session to be already established via **tpm2_startauthsession**(1). If
+the input session is a trial session this tool generates a policy digest that 
+associates a signing authority's public key name with the policy being 
+authorized to run. If the input session is a policy session tpm2_policyauthorize
+looks for a verification ticket from the TPM to attest that the TPM has verified
+the signature on the policy digest.
+
+# OPTIONS
+
+  * **-f**, **--policy-file**=_POLICY\_FILE_:
+    File to save the policy digest.
+
+  * **-S**, **--session**=_SESSION_FILE_:
+
+    The policy session file generated via the **-S** option to
+    **tpm2_startauthsession**(1).
+
+  * **-a**, **--input-policy-file**=_POLICY\_FILE_:
```

Comment:
This comes from -f in tpm2_startauthsession, shouldn't we make the input -f to correlate these things?

We can change the policy-file output to either -o, or can we clobber the one they give us?